### PR TITLE
Prepare the library for being published and installed from npmjs

### PR DESCRIPTION
**Pull request by bzvestey**

### CODE REVIEW

File: src/AssistantAPIClient.js [hunk 1]
```diff
@@ -48,15 +48,19 @@ class AssistantAPIClient{
   // Services
   //
   getServiceById = async (id) =>{
-    return await APICall('getServiceById', id)
+    var res = await APICall('getServiceById', id)
+    // Call function on returned object--call by reference to parent
+    console.log('res',res)
```
Comment (jcrosetto-maana) [inline src/AssistantAPIClient.js]: console.log

File: babel.config.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const presets = [
+  [
+    "@babel/env",
+    {
+      targets: {
+        edge: "17",
+        firefox: "60",
+        chrome: "67",
+        safari: "11.1",
```
Comment (jcrosetto-maana) [inline babel.config.js]: how were these determined?

File: src/AssistantAPIClient.js (hunk 1 above)
Comment (bzvestey) [reply src/AssistantAPIClient.js]: This is intentional at the moment, but it should be marked for being deleted.

File: babel.config.js (hunk 2 above)
Comment (bzvestey) [reply babel.config.js]: These are currently just the default that was given to me by the tutorial for adding babel.  Do we have a specific selection for kportal that we use?  @LoganGore please make sure we update it based on what @jcrosetto-maana says is required.

**Pull request merged**